sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -799,6 +812,12 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
         msalLogger.verbose("Attempting broker authentication without the default broker account");
       }
 
+      if(options.proofOfPossessionOptions){
+        (interactiveRequest as any).shrNonce = options.proofOfPossessionOptions.nonce;
```

Comment:
If the types are fixed up in msal-node can we remove the `as any` cast?

Reply:
done